pkg/pool-stable/contracts/StableMath.sol

```diff
@@ -28,6 +28,7 @@ contract StableMath {
 
     uint256 internal constant _MIN_AMP = 1;
     uint256 internal constant _MAX_AMP = 5000;
+    uint256 internal constant _AMP_PRECISION = 1e4;
```

Comment:
Curve has it in 1e2 (100), However,I think we can leave it at 1e4 (10000) and tests if there are no overflows for 5 tokens pool.